src/snovault/schema_views.py

```diff
@@ -65,3 +69,30 @@ def schemas(context, request):
         subtypes[name] = type_info.subtypes
 
     return schemas
+
+
+@view_config(route_name='schemas_map', request_method='GET',
+             decorator=etag_app_version_effective_principals)
+def schemas_map(context, request):
+    """Returns a mapping of schema object name to the corresponding schema ID.
```

Comment:
No need for comments like these in the python code.  We don't do that.  Please remove.  

Effort should be directed at writing python code that doesn't need comments.  Which I think the function does fine.  Comments end up repeating what someone could glean from just looking at the code.

Reply:
https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments

“A function must have a docstring, unless it meets all of the following criteria:”

I think this function meets the second, but not the other two.